spec/classes/lightblue/application/datamgmt_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe 'lightblue::application::datamgmt' do
+  let(:hiera_config){ 'spec/fixtures/hiera/hiera.yaml' }
+
+  let :facts do
+    {
+      :architecture => 'x86_64',
+      :osfamily => 'RedHat',
+      :operatingsystemrelease => '7.0'
+    }
+  end
+
+  context 'with default params' do
+    it do
+      should contain_package('lightblue-data-mgmt') \
+        .with_ensure('latest') \
+        .with_require(['Class[Lightblue::Yumrepo::Lightblue]', 'Class[Lightblue::Eap]'])
+    end
+
+    it do
```

Comment:
Put white space between them then.  I expect that each `it` block is re-executing the test which is just going to add to test execution time and is unnecessary overhead.  There is nothing different between each `it` section you've defined, so don't run the test many times.  You're essentially splitting asserts across many test cases with the test conditions being identical.